themes/acquia_claro/acquia_claro.theme

```diff
@@ -7,6 +7,25 @@
 
 use Drupal\acquia_claro\PreRender;
 
+/**
+ * Prepares variables for install page templates.
+ *
+ * Default template: install-page.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - content - An array of page content.
+ *
+ * @see template_preprocess_maintenance_page()
+ */
+function acquia_claro_preprocess_install_page(array &$variables) {
+  $variables['drupal_core_version'] = \Drupal::VERSION;
+  $variables['#attached']['library'][] = 'seven/install-page';
+  $variables['#attached']['library'][] = 'acquia_claro/install-page';
+  $acquia_cms_path = \Drupal::service('extension.path.resolver')->getPath('profile', 'acquia_cms');
+  $variables['install_page_logo_path'] = \Drupal::service('file_url_generator')->generateAbsoluteString($acquia_cms_path . '/acquia_cms.png');
```

Comment:
@panshulK Let's not use `file_url_generator` service and see if there's any other alternative. Because this service came in Drupal 9.3 and is not available in Drupal 9.2.x.(https://api.drupal.org/api/drupal/9.2.x/search/file_url_generator)

Reply:
Done